root_pandas/readwrite.py

```diff
@@ -169,14 +170,29 @@ def genchunks():
     return convert_to_dataframe(arr)
 
 
+
+
+
 def convert_to_dataframe(array):
-    indices = list(filter(lambda x: x.startswith('__index__'), array.dtype.names))
+
+    def get_nonscalar_columns(array):
```

Comment:
But wouldn't that mean that the function gets exposed on the root_pandas namespace?
Since the function is only needed in this one place, maybe it would be better to leave it there?

I am not having a strong opinion on this, just trying to understand.